test/rihanna_test.exs

```diff
@@ -136,6 +136,33 @@ defmodule RihannaTest do
       end
     end
 
+    test "`enqueue/2`, `get_job_by_id/1`, `delete/1` using Ecto Repo passed in the options", %{
+      pg: pg
+    } do
+      opts = [producer_postgres_connection: {Ecto, TestApp.Repo}]
```

Comment:
Wouldn't this pass even if it's usign the default?

Reply:
Isn't the default tested here?   
https://github.com/samphilipd/rihanna/blob/82f3c9a7f51db9620e0527c3d3a7c0ef67bd2bb1/test/rihanna_test.exs#L113

or I didn't get the question right...

Reply:
I mean, how can we be sure it's using the passed repo and not the default one from the config?

Reply:
Ok, I see.
The other tests are wrapped in this:
`TemporaryEnv.put :rihanna, :producer_postgres_connection, {Ecto, TestApp.Repo} do`
or this
`TemporaryEnv.put :rihanna, :producer_postgres_connection, {Postgrex, pg}`

This one is not. Making the `opts = []` will fail the test. 
Isn't that the proof that it's using the repo from the options?

I could add: `refute Rihanna.Config.producer_postgres_connection() == {Ecto, TestApp.Repo}` before the transaction

Would you recommend any additional check?

Reply:
Aaah yes I see. Yup the `refute` would make it more explicit